client/asset/dcr/dcr.go

```diff
@@ -3397,6 +3397,77 @@ func (dcr *ExchangeWallet) signTxAndAddChange(baseTx *wire.MsgTx, feeRate uint64
 	return msgTx, change, changeAddr, lastFee, nil
 }
 
+// EstimateSendTxFee returns a tx fee estimate for sending or withdrawing the
+// provided amount using the provided feeRate.
+func (dcr *ExchangeWallet) EstimateSendTxFee(sendAmount, feeRate uint64, subtract bool) (fee uint64, err error) {
+	if sendAmount == 0 {
+		return 0, fmt.Errorf("cannot check fee: send amount= 0")
```

Comment:
typo: `amount= 0` -> `amount = 0`,

also, could be more specific and instead of `cannot` say something like `it doesn't make sense to`